canonical/ubuntu.com
# CUE Scheduling endpoint change

## Done

- Scheduling and Cancelling exams changed to CS endpoints.
## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- Purchase activation keys from the back-office tool
    - Activate those keys by going to https://ubuntu-com-12295.demos.haus/credentials/redeem/{activationKey}
    - Go to https://ubuntu-com-12448.demos.haus/credentials/your-exams to verify that your keys have been activated
- Try scheduling an exam
- Now try to cancel that exam
- et voilà !

## Issue / Card

https://warthogs.atlassian.net/browse/CRED-84

## Screenshots

[If relevant, please include a screenshot.]


## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
